lib/srv/regular/sshserver_test.go

```diff
@@ -1481,8 +1528,8 @@ func TestLimiter(t *testing.T) {
 
 	// current connections = 2
```

Comment:
@EdwardDowling I looked at bit into this test and I think that the numbers in these comments are all off which makes it very confusing. For example, above it says "current connections = 1 (one connection is opened from SetUpTest)" which I don't think is true - because we're creating a new SSH server with a new limiter. Also, it says "maxConnections =3" but it's actually 2. Could you fix up these comments so they're accurate while you're here?